DBN_article/DBN_article.md

```diff
@@ -0,0 +1,187 @@
+unexplored data. A typical RBM architecture is represented below 
+(where h represents hidden nodes and v – visible nodes). \
+
+<div style="text-align: center">
+    <img src="dbn_2.png" alt="RBM architecture" style="width:400px;"/>
+</div>
+
+&nbsp; &nbsp; &nbsp;Coming back to DBNs, all RBMs it consists of, are trained in an 
+unsupervised manner, one at a time, the output of one of them 
+being the input for the next one. The output of the final machine 
+is used either in classification or regression tasks, making the 
+general DBN architecture look like the one represented below. 
+The prior reason that lead to the appearance of DBNs is to create 
+unbiased values stored in leaf nodes and to avoid being stuck in 
+the local minima. 
+
+<div style="text-align: center">
+    <img src="dbn_3.jpg" alt="DBN architecture" style="width:400px;"/>
+</div>
+
+## DBNs overview
+
+&nbsp;&nbsp;&nbsp;However, besides the Standard DBNs described earlier, there might be 
+distinguished several extensions that incorporate different 
+structures and components. Therefore, the following are the most 
+notorious variations of DBNs:
+* Convolutional Deep Belief Networks (ConvDBNs) – in addition to RBMs, 
+incorporates convolutional layers;
+* Temporal Deep Belief Networks (Temporal DBNs) – extended Standard DBN 
+to model sequential and time-series data, incorporating 
+recurrent connections or temporal dependencies between layers;
+* Variational Deep Belief Networks (VDBNs) – use a probabilistic 
+modeling technique, variational interface, in DBMs, making 
+multiple layers of hidden units fully connected between 
+consecutive layers;
+* *Stacked Autoencoders -  although not a DBN in the traditional 
+sense, when multiple autoencoders are stacked, they form a deep 
+architecture that shares a considerate amount of similarities 
+with Deep Belief Networks.
+
+## Math base
+&nbsp;&nbsp;&nbsp;Despite the diversity of the existing DBNs, it shouldn’t be 
+forgotten the fact that all of the share similar “roots”. 
+Recall that DBN is a network assembled out of many single 
+networks. Except the first and last layers, others play dual 
+role serving at the same time as hidden layers that comes before 
+and as input for the following one [1] . The joint distribution 
+between the observed vector X and the hidden layers hk may be 
+expressed using the formula: \
+
+<div style="text-align: center">
+    <img src="dbn_4.png" alt="Base formula" style="width:350px;"/>
+</div>
+
+, where:
+* X = h_0,
+* P(h<sup>k</sup>|h<sup>k+1</sup>) – a conditional distribution for the visible units 
+conditioned on the hidden units of the RBM at level k,
+* P(h<sup>l-1</sup>, h<sup>l</sup>) – visible-hidden joint distribution in the top-level RBM.
+
+## DBNs Training
+
+&nbsp;&nbsp;&nbsp;Before training a DBN, it is necessary to remember that every 
+individual RBM should be trained. To keep it brief, the 
+next steps need to be gone through to train the described model:
```

Comment:
The following explanation sounds really short and concise could maybe add an implementation example around it as well